roles/add-sonar-plugin/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Get JAR name
+  set_fact:
+    jar_name: "{{ plugin_url.split('/')[-1] }}"
+
+- name: Download the SonarQube plugin
+  get_url:
+    url: "{{ plugin_url }}"
+    dest: "/tmp/{{ jar_name }}"
+    mode: 0440
+
+## The Jinja2 template and the Go template in the following task conflict,
+## so the weird double-curly-braces surrounded by double-curly-braces is required.
+- name: Find sonarqube pod
+  command: 'oc get pods --namespace={{ namespace }} -l="app==sonarqube" -o go-template="{{ "{{" }}range .items{{ "}}" }}{{ "{{" }}.metadata.name{{ "}}" }}{{ "{{" }}end{{ "}}" }}"'
```

Comment:
This line could be simplified to significantly improve readability, also recommend using a multiline yaml quote to avoid the long line. For example:
```
    command: >-
      oc get pods --namespace={{ namespace }} -l="app==sonarqube"
      -o jsonpath="{.items[0].metadata.name}"
```